robotoff/workers/queues.py

```diff
@@ -1,23 +1,58 @@
+import random
 import threading
 import time
 from typing import Callable, Optional
 
 from rq import Queue
 from rq.job import Job
 
+from robotoff import settings
 from robotoff.redis import redis_conn
-from robotoff.types import WorkerQueue
+from robotoff.types import ProductIdentifier
+from robotoff.utils import get_logger
 
-high_queue = Queue(WorkerQueue.robotoff_high.value, connection=redis_conn)
-low_queue = Queue(WorkerQueue.robotoff_low.value, connection=redis_conn)
+logger = get_logger(__name__)
+high_queues = [
+    Queue(f"robotoff-high-{i+1}", connection=redis_conn)
+    for i in range(settings.NUM_RQ_WORKERS)
+]
```

Comment:
NUM_RQ_WORKERS is 4 but you have 2 services ?

Reply:
We have 4 services, no? `low_{1,2}` and `high_{1,2}`

Reply:
yes but here it's only `robotoff-high-`, isn't it ?

Reply:
ok, my bad, I didn't see low queues where also listening for high queues.

Why aren't all worker listening the low queue then ?

I don't really understand your pattern of naming some high / other low, while they still have a high queue to listen…

Are there some tasks that you only dedicate to worker_high_1/2 ?

Reply:
Only low priority workers are listening to low queues. The idea was that all workers handle high priority jobs first, but only a limited number of workers (=2) can handle low priority jobs, so that the server is not stressed (as we can have 1000s of low priority jobs when launched using CLI).